docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Example scenario for data warehousing and analytics in Azure
+description: Use a data warehouse in Azure to consolidate data from multiple sources and optimize data analytics.
+author: alexbuckgit
+ms.date: 08/16/2018
+---
+
+# Example scenario for data warehousing and analytics in Azure
+
+Build a data pipeline to integrate large amounts of data from multiple sources into a cloud-scale analytics platform. This scenario is relevant for:
```

Comment:
I like the idea, but let's make it more of a conversation since it's the first thing you're reading.  My suggestion would be to reword it to:

This example scenario builds a data pipeline to integrate large amounts of data from multiple sources into a single cloud-scale analytics platform.  This specific scenario was modeled from a sales and marketing company, but the design patterns would be similar for many industries requiring advanced analytics of large datasets including e-commerce, retail, and healthcare.